th2_data_services/interfaces/events_tree/events_tree_collection.py

```diff
@@ -247,12 +247,32 @@ def get_root_by_id(self, id) -> EventsTree:
 
         Raises:
             EventIdNotInTree: If event id is not in the trees.
+
```

Comment:
Please remove this comment because this issue will be fixed in the https://exactpro.atlassian.net/browse/TH2-3593
It's possible that this comment will be automatically added during merge and no one will note it.